operator/builtin/input/windows/event.go

```diff
@@ -0,0 +1,79 @@
+// +build windows
+
+package windows
+
+import (
+	"fmt"
+)
+
+// Event is an event stored in windows event log.
+type Event struct {
+	handle uintptr
+}
+
+// RenderSimple will render the event as EventXML without formatted info.
+func (e *Event) RenderSimple(buffer Buffer) (EventXML, error) {
+	if e.handle == 0 {
+		return EventXML{}, fmt.Errorf("event handle does not exist")
+	}
+
+	var bufferUsed, propertyCount uint32
+	err := evtRender(0, e.handle, EvtRenderEventXML, buffer.Size(), buffer.FirstByte(), &bufferUsed, &propertyCount)
+	if err == ErrorInsufficientBuffer {
+		buffer.UpdateSize(bufferUsed)
+		return e.RenderSimple(buffer)
+	}
+
+	if err != nil {
+		return EventXML{}, fmt.Errorf("syscall to 'EvtRender' failed: %s", err)
+	}
+
+	bytes, err := buffer.ReadBytes(bufferUsed)
+	if err != nil {
+		return EventXML{}, fmt.Errorf("failed to read bytes from buffer: %s", err)
+	}
+
+	return unmarshalEventXML(bytes)
+}
+
+// RenderFormatted will render the event as EventXML with formatted info.
+func (e *Event) RenderFormatted(buffer Buffer, publisher Publisher) (EventXML, error) {
+	var bufferUsed uint32
```

Comment:
Is there a reason we don't have a `e.handle == 0` check here?

Reply:
No. This was an accidental miss. It should be added.